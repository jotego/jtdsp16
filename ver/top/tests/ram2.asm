r0=0x100
r1=0x23
r2=0xbabe
r3=0xa055
j=3
# Test ++ and --
*r0++j=r1
*r0++j=r2
*r0=r3
r0=0x100
rb=*r0++j
re=*r0
end:
goto end
