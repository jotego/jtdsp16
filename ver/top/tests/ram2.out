r0=0x0103
r1=0x0023
r2=0xbabe
r3=0xa055
rb=0x0023
re=0xbabe
j =0x0003
k =0x0000
pr=0x0000
pt=0x0000
i =0x0000
a0=0x000000000
a1=0x000000000
x=0x0000
y=0x00000000
p=0x00000000
c0=0x0000
c1=0x0000
c2=0x0000
auc=0x0000
psw=0x0000
